cmd/edit/machinepool/cmd.go

```diff
@@ -166,8 +166,16 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 
 	if cluster.Hypershift().Enabled() {
```

Comment:
@chenz4027 As a suggestion to avoid the repeated `if err` check:

```
var err error

if cluster.Hypershift().Enabled() {
 err = editNodePool(cmd, machinePoolID, clusterKey, cluster, r)
else
 err = editMachinePool(cmd, machinePoolID, clusterKey, cluster, r)
}

if err != nil {
 r.Reporter.Errorf("%s", err)
 os.Exit(1)
}

```

Reply:
Thanks @robpblake , will refactor it

Reply:
Updated